src/status_im2/subs/root.cljs

```diff
@@ -100,6 +100,9 @@
 (reg-root-key-sub :chat/inputs :chat/inputs)
 (reg-root-key-sub :chat/memberships :chat/memberships)
 (reg-root-key-sub :camera-roll-photos :camera-roll-photos)
+(reg-root-key-sub :camera-roll-end-cursor :camera-roll-end-cursor)
```

Comment:
Not suggesting anything here, but does anybody know if adding more root keys to the graph can meaningfully impact performance?

Is it generally better to have less root keys? Would it better for the performance of the app to create just one `:camera-roll` root key and have the `next-page` and `loading-more` keys inside it?

My intuition tells me that it's better to have less root keys for things that are interconnected, so that we can better leverage subscription caching.

@flexsurfer cc

Reply:
yes i just had the same feeling lately reviewing PRs,  we should group props in root keymap

Reply:
would be cool to add this to guidelines